docs/repositories-configure/integrations/github-integration.md

```diff
@@ -48,3 +50,18 @@ Adds annotations on the lines of the pull request where Codacy finds new issues.
 Shows an overall view of the changes in the pull request, including new issues and metrics such as complexity and duplication. To enable this option, you must enable **Status checks** first.
 
 ![Pull request summary on GitHub](images/github-integration-pr-summary.png)
+
+### Suggest fixes
+
+Adds comments on the lines of the pull request where Codacy finds new issues with suggestions on how to fix the issues. Codacy doesn't make any changes automatically. To apply the changes, manually review and accept the suggestions.
```

Comment:
Maybe apply makes more sense here?
```suggestion
Adds comments on the lines of the pull request where Codacy finds new issues with suggestions on how to fix the issues. Codacy doesn't apply any changes automatically. To apply the changes, manually review and accept the suggestions.
```

Reply:
Yes, as it ties in better with the next sentence about manually applying the changes. Nice suggestion! :+1: